scala/build/Dockerfile.maintenance

```
@@ -0,0 +1,266 @@
+# sbt build
+RUN whoami
+RUN env
+RUN pwd;ls -all
+RUN sbt sbtVersion
+RUN yarn --version
+RUN python3 --version
+RUN aws --version
+RUN sudo aws --version
+RUN ls -all /home/qa
+RUN cat /home/qa/package.json
+RUN sudo chmod 4755 /bin/ping
```

Hadolint

[error] Do not use sudo as it leads to unpredictable behavior. Use a tool like gosu to enforce root

(DL3004)